ckanext/recombinant/templates/recombinant/snippets/choices_reference.html

```diff
@@ -1,4 +1,4 @@
-<h3>{{ _("Field choices reference") }}</h3>
```

Comment:
@TkTech do you remember why the (required) block was added here? https://github.com/open-data/ckanext-canada/commit/aa297a6745ebe241fbdde7f6429b206f059d9d40#diff-3d149a3f59f2d696b95fa3a4927348b1R6372